components/bookings/BookingForm.vue

```diff
@@ -183,6 +116,11 @@ nl:
             </div>
           </div>
         </div>
+
+        <p class="mt-4 px-3 py-3 rounded shadow">
```

Comment:
Cool!

```suggestion
        <p class="mt-4 px-3 py-3 rounded shadow text-gray-500">
```
makes it a bit more subtle